<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>BalanceDialog</class>
 <widget class="QDialog" name="BalanceDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>400</width>
    <height>359</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <widget class="QWidget" name="formLayoutWidget">
   <property name="geometry">
    <rect>
     <x>70</x>
     <y>150</y>
     <width>271</width>
     <height>71</height>
    </rect>
   </property>
   <layout class="QFormLayout" name="formLayout">
    <item row="0" column="0">
     <widget class="QLabel" name="labelDailyBalance">
      <property name="text">
       <string>Saldo dzienne:      </string>
      </property>
     </widget>
    </item>
    <item row="1" column="0">
     <widget class="QLabel" name="labelMonthlyBalance">
      <property name="text">
       <string>Saldo miesięczne: </string>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QWidget" name="formLayoutWidget_2">
   <property name="geometry">
    <rect>
     <x>70</x>
     <y>110</y>
     <width>271</width>
     <height>31</height>
    </rect>
   </property>
   <layout class="QFormLayout" name="formLayout_2">
    <item row="0" column="1">
     <widget class="QSpinBox" name="waiterIdSpinBox"/>
    </item>
    <item row="0" column="0">
     <widget class="QLabel" name="labelWaiterId">
      <property name="text">
       <string>ID kelnera: </string>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QPushButton" name="btnLoadBalance">
   <property name="geometry">
    <rect>
     <x>140</x>
     <y>230</y>
     <width>101</width>
     <height>32</height>
    </rect>
   </property>
   <property name="text">
    <string>Pokaż saldo</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
